pubspec.yaml

```diff
@@ -6,10 +6,6 @@ version: 0.0.1+1
 environment:
   sdk: ">=2.7.0 <3.0.0"
 
-dependency_overrides:
```

Comment:
The only change I made was to remove the hack for firebase messaging that we added as it's no longer needed

Reply:
shweet :)